packages/core/src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -341,4 +353,32 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
             }
         }, 0);
     }
+
+    private async _loadPaymentMethodsAllowed(mollie: MolliePaymentInitializeOptions, methodId: string, gatewayId: string, state: InternalCheckoutSelectors){
```

Comment:
We seem to calling this method in two places, is there a case when this is executed both times? Then do we have two elements on the page with the same `id`?

Reply:
There is no case where it is executed two times, the execution of line 101 is only when there is a change in the coupons or in the balance of the cart.